src/parser.djs

```diff
@@ -68,6 +69,16 @@ such isInlineWhitespace much chr
     wow
 wow result
 
+quiet
+  Determines if the first character of the content is a new line character
+  
+  @return true if the content starts with a new line character, false otherwise
+loud
+such startsWithNewLine much content
```

Comment:
I'm not sure \r should count as a newline, are there issues with just including it as inlineWhitespace?

Reply:
No there weren't! It ends up correctly consuming the `\'r'` at the end of lines now as well and not throwing parse errors!